pyramid_okta/utils.py

```diff
@@ -186,61 +202,96 @@ def clear_session(session_id):
     else:
         return True
 
+def get_groups(filter='', limit=None, app_id=None):
```

Comment:
correct, we'll just use okta directly and use group policies -- however, okta does have the ability to nest groups (kinda).  You can setup rule based groups, ie, add users of XYZ deparment,division,title to this group so they do not need to be manually maintained.